api/v1beta1/verticadb_types.go

```diff
@@ -1312,6 +1329,28 @@ func (v *VerticaDB) IsAgentEnabled() bool {
 }
 
 // IsAdditionalConfigMapEmpty returns true if there is no extra
+// config parameters.
 func (v *VerticaDB) IsAdditionalConfigMapEmpty() bool {
 	return len(v.Spec.Communal.AdditionalConfig) == 0
 }
+
+// IsDepotVolumeEmptyDir returns true if the depot volume's type
+// is emptyDir.
+func (v *VerticaDB) IsDepotVolumeEmptyDir() bool {
+	return v.Spec.Local.DepotVolume == EmptyDir
+}
+
+// IsDepotVolumePersistentVolume returns true if the depot volume's type
+// is persistentVolume.
+func (v *VerticaDB) IsDepotVolumePersistentVolume() bool {
+	return v.Spec.Local.DepotVolume == PersistentVolume
```

Comment:
I followed the way `InitPolicy` has been defined. The default value is `PersistentVolume` so if `depotVolume` is not set in the CR, this value will be assumed; if it is set to empty string, it will be rejected.